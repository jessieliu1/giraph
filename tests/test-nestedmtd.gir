node makenode(int a){
   node r;
   r.set_data(a);
   return r;
}

digraph makegraph(int a,int b){
   digraph g = [A:a <-> B:b];
   return g;
}

int main(){
   node n = makenode(1);
   print(makenode(450).data());
   for_edge(e : makegraph(makenode(39).data(), 5) ){
        print(e.to().data() + n.data());
   }

return 0;
}
