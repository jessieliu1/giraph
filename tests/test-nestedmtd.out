450
6
40
